code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -137,6 +138,12 @@
 	gas_type = /datum/gas/water_vapor
 	filled = 1
 
+/obj/machinery/portable_atmospherics/canister/miasma
+	name = "miasma canister"
+	desc = "Miasma. Makes you wish your nose were blocked."
```

Comment:
Try `Makes you wish your nose was blocked.` The `were` in the existing line sounds a bit awkward

Reply:
You might be right